stdlib/prelude.kl

```diff
@@ -0,0 +1,23 @@
+#lang "n-ary-app.kl"
+
+[define id [lambda (x) x]]
+[define const [lambda (x y) x]]
+[define compose [lambda (f g x) (f (g x))]]
+(define compose* (flet (comp (f g x) (f (g x))) comp))
+
+(export #%module #%app
```

Comment:
I actually think that `"prelude.kl"` should export the contents of `n-ary-app` again at phase 1. That is, it should import it shifted, and then repeat the same export list shifted again. The nice thing about this is that it makes them available in macros as well without any extra ceremony (kind of like how `#lang racket` exports the contents of `racket/base` at phase 1).

But this is something that can easily be added later!

Reply:
Yes, let's go ahead and do this in a follow-up. I mostly moved these files here as a kind of proof-of-concept that importing the stdlib would work correctly. In another PR, we can make it actually useful.